pipeline_0_annotate.sh

```diff
@@ -1,39 +1,20 @@
 #!/bin/bash
+ANNOTATE_OUTPUT_DIR="${OUTPUT_DIR}"/step0_annotate
+
 VEP_INPUT="${INPUT}"
-VEP_OUTPUT_DIR="${OUTPUT_DIR}"/step0_vep
+VEP_OUTPUT_DIR="${ANNOTATE_OUTPUT_DIR}"/step0a_vep
 VEP_OUTPUT="${VEP_OUTPUT_DIR}"/"${OUTPUT_FILE}"
 VEP_OUTPUT_STATS="${VEP_OUTPUT}"
+VEP_OUTPUT_ERRORS="${VEP_OUTPUT}.err"
 
 mkdir -p "${VEP_OUTPUT_DIR}"
 
-if [ -f "$VEP_OUTPUT" ]
-then
-        if [ "$FORCE" == "1" ]
-        then
-                rm "$VEP_OUTPUT"
-        else
-                echo "$VEP_OUTPUT already exists, use -f to overwrite.
-                "
-                exit 2
-        fi
-fi
-if [ -f "$VEP_OUTPUT_STATS" ]
-then
-        if [ "$FORCE" == "1" ]
-        then
-                rm "$VEP_OUTPUT_STATS"
-        else
-                echo "$VEP_OUTPUT_STATS already exists, use -f to overwrite.
-                "
-                exit 2
-        fi
-fi
-
 module load VEP
 
 vep \
 --input_file ${VEP_INPUT} --format vcf \
 --output_file ${VEP_OUTPUT} --vcf --compress_output bgzip --force_overwrite \
+--warning_file ${VEP_OUTPUT_ERRORS} \
 --stats_file ${VEP_OUTPUT_STATS} --stats_text \
 --offline --cache --dir_cache /apps/data/Ensembl/VEP/100 --fasta /apps/data/Ensembl/VEP/100/Homo_sapiens.GRCh37.75.dna.primary_assembly.fa.gz \
```

Comment:
we can't use ```"${ASSEMBLY}"``` here because we don't have GRCh38 resources available. add a code comment to update this in future?

Reply:
The fact that te resources aren't there is not directly related to the pipeline code right? I'd opt for using the ${ASSEMBLY}